psga/config/pipeline-schema.yaml

```
@@ -0,0 +1,95 @@
+pipeline:
+  name: sars-cov-2
+  docker_image: ''
+  protocols:
+  - name: illumina-sars-cov-2-artic-v3
+    kit:
+      name: Artic V3
```

@pdp10 You mentioned that these values are wrong. Can you please suggest the correct value. I believe it was something along the lines of: 

```suggestion
      name: Artic_V3
```

---

@pdp10 that's correct, all of the values that we are storing are for the purpose of being able to call the pipeline.

---

Values can be (case sensitive):

* `ARTIC_V${n}` with n in `["1", "2", "3", "4", "4.1"]`
* `Midnight-IDT_V1`
* `Midnight-ONT_V${n}` with n in `["1", "2", "3"]`
* `NEB-VarSkip_v${n}` with n in `["1a", "1a-long", "2", "2b"]`    (note that `v` is lower case here)